lib/community/digest.js

```diff
@@ -312,7 +335,7 @@ Digest.test = function (communityId, timeAmount, timeUnit, userId) {
   var now = moment()
   var then = moment().subtract(timeAmount, timeUnit)
   return Community.find(communityId).then(community => {
-    var digest = new Digest(community, then, now)
+    var digest = new Digest(community, then, now, "daily")
```

Comment:
btw, https://codeclimate.com/github/Hylozoic/hylo-node/lib/community/digest.js is interesting if you haven't seen it yet